pkgs/by-name/pr/prometheus-dnssec-exporter/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub, }:
+buildGoModule rec {
+  pname = "prometheus-dnssec-exporter";
+  version = "0-unstable-2023-03-05";
+
+  src = fetchFromGitHub {
+    owner = "chrj";
+    repo = "prometheus-dnssec-exporter";
+    rev = "b638685ed8d5919a88b45e85b3aec702f0fcc393";
+    hash = "sha256-SGoQKSgTRfSyA65xEZ9P7Z956sLMhB88h3HaXmFywiQ=";
+  };
+
+  vendorHash = "sha256-u7X8v7h1aL8B1el4jFzGRKHvnaK+Rz0OCitaC6xgyjw=";
+
+  doCheck = false;
```

Comment:
Running https://github.com/chrj/prometheus-dnssec-exporter/blob/master/main_test.go does not work?

Reply:
Good point. I must confess that I had probably mistakenly taken that from another package. I'll check it out.

Reply:
Of course they worked as expected!